packages/forms/src/elements/file-list/components/Delete.tsx

```diff
@@ -5,32 +5,40 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { HTMLAttributes } from 'react';
+import React, { ButtonHTMLAttributes } from 'react';
 import { composeEventHandlers } from '@zendeskgarden/container-utilities';
 import TrashIconCompact from '@zendeskgarden/svg-icons/src/12/trash-stroke.svg';
 import TrashIconDefault from '@zendeskgarden/svg-icons/src/16/trash-stroke.svg';
 import useFileContext from '../../../utils/useFileContext';
 import { StyledFileDelete } from '../../../styled';
 
-const DeleteComponent = React.forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
-  (props, ref) => {
-    const fileContext = useFileContext();
-    const onMouseDown = composeEventHandlers(
-      props.onMouseDown,
-      (event: MouseEvent) => event.preventDefault() // prevent parent File focus
-    );
+const DeleteComponent = React.forwardRef<
+  HTMLButtonElement,
+  ButtonHTMLAttributes<HTMLButtonElement>
+>((props, ref) => {
+  const fileContext = useFileContext();
+  const onMouseDown = composeEventHandlers(
+    props.onMouseDown,
+    (event: MouseEvent) => event.preventDefault() // prevent parent File focus
+  );
 
-    return (
-      <StyledFileDelete ref={ref} {...props} onMouseDown={onMouseDown}>
-        {fileContext && fileContext.isCompact ? <TrashIconCompact /> : <TrashIconDefault />}
-      </StyledFileDelete>
-    );
-  }
-);
+  return (
+    <StyledFileDelete
+      ref={ref}
+      aria-label="Press delete to remove"
```

Comment:
Should this be a `useText` string?

Reply:
Fixed: https://github.com/zendeskgarden/react-components/pull/1509/commits/54c788cd319976a36af608e55fa1ee55adb3e4bc